server/src/__tests__/utils/environment.ts

```diff
@@ -0,0 +1,49 @@
+import { createMockClubEntities } from './create-club-entities'
+import { createMockEventEntities } from './create-event-entities'
+import { createMockTagEntities } from './create-tag-entities'
+import { setup, types } from '../../app/'
+
+const clubEntities = createMockClubEntities()
+const eventEntities = createMockEventEntities()
+const tagEntities = createMockTagEntities()
+
+export interface Variables {
+  app: types.ExpressApp
+}
+
+// This is similar to Jest environments.
+// We don't use Jest environments here because they modify the global object in a type-unsafe way.
+class TestEnvironment {
+  protected entities!: types.Entities
+  protected orm!: types.MikroORM
+  protected mikroEntityRepos!: types.MikroEntityRepos
+
+  public async setup(): Promise<Variables> {
+    const entities = setup.Entities()
+    this.orm = await setup.MikroORM({ debug: false })
+    this.mikroEntityRepos = setup.MikroEntityRepos(entities, this.orm)
+
+    eventEntities.forEach((eventEntity) => {
+      eventEntity.clubs.add(...clubEntities)
+      eventEntity.tags.add(...tagEntities)
+    })
+    clubEntities.forEach((clubEntity) => clubEntity.tags.add(...tagEntities))
+    await this.mikroEntityRepos.event.persist(eventEntities).flush()
```

Comment:
This is here to match the existing `populate/teardown` implementation, but in the long run we shouldn't do this. Jest supports _global_ setup/teardown. By moving this expensive operation into the global setup, we would persist/remove the objects once per run instead of n times (where n is the number of test files).